src/util/git.js

```diff
@@ -293,12 +295,15 @@ export default class Git {
           parser.on('finish', done);
 
           parser.on('entry', (header, stream, next) => {
-            let string = '';
+            const decoder = new StringDecoder('utf8');
+            let fileContent = '';
+
             stream.on('data', (buffer) => {
-              string += buffer.toString();
+              fileContent += decoder.write(buffer);
             });
             stream.on('end', () => {
-              update(string);
+              fileContent += decoder.end();
+              update(fileContent);
```

Comment:
decoded.end() won't return a full string.
It would just return remaining bytes if there were any.
So I needed to accumulate them in a string